src/helpers.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable import/no-cycle */
+import Utils from './utils.js';
+
+export default class Helper {
+  /**
+   * @function formHandler - instance method for handling of form submission
+   */
+  static formHandler(e) {
+    e.preventDefault();
+    Utils.add();
+    Utils.clear();
+    Helper.checkHandler();
+    Helper.handleFocus();
```

Comment:
I mean there is nothing to mention here :sunglasses: .. you code is almost perfect :ok_hand: 

[DRY] Could we put those two statements in a function and execute it also instead of line 25 , 26 :thinking: